example/src/main/scala/example/HelloWorldAdvanced.scala

```diff
@@ -32,9 +32,9 @@ object HelloWorldAdvanced extends App {
 
     // Create a new server
     server.make
-      .use(_ =>
+      .use(bootstrap =>
```

Comment:
@tusharmath @amitksingh1490  added this just as an example of how we are supporting this feature.